<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransactiveAboutBox</class>
 <widget class="QMainWindow" name="TransactiveAboutBox">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestQt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>263</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="projectName">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Project Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="copyright">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Copyright Â© 2003 Ripple System Pty Ltd. All rights reserved.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="version">
       <property name="text">
        <string>Version:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="component">
       <property name="text">
        <string>Component:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="buildDate">
       <property name="text">
        <string>Build Date:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>471</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="releaseNumber">
       <property name="text">
        <string>Release number</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="applicationName">
       <property name="text">
        <string>Application name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="buildDateTime">
       <property name="text">
        <string>Build Date/Time</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>280</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
